backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -133,14 +124,14 @@ public IActionResult UpdateUser([FromBody] Model.UserModel model)
         [HttpDelete("{id}")]
         public IActionResult DeleteUser(Guid id, [FromBody] Model.UserModel model)
```

Comment:
I'm not happy with how the rowversion concurrency checking is working here, especially since the rowversion needs to be copied again in the service. If you don't have any ideas on how to clean this up I'll try and find some examples.

Reply:
The workflow is this;
- request object (has rowversion)
- submit object (include the rowversion)
- EF will compare the rowversion in the db with your submitted object and throw an error if they are not the same.

Be aware that EntityFramework Core has a issue with byte[] values (i.e. rowversion) and you have to copy the array into the existing array rather than just make the entity.RowVersion = rowversion.

AutoMapper should already be able to convert the string to a byte[] and vise-versa.  Just be aware of the above statement.